packages/ember-template-compiler/index.ts

```diff
@@ -37,7 +37,7 @@ export {
 export { RESOLUTION_MODE_TRANSFORMS, STRICT_MODE_TRANSFORMS } from './lib/plugins/index';
 export { EmberPrecompileOptions } from './lib/types';
 
-export { preprocess as _preprocess, print as _print } from '@glimmer/syntax';
+export { preprocess as _preprocess, print as _print, getTemplateLocals } from '@glimmer/syntax';
```

Comment:
All of glimmer syntax is exported (just below), let's just rely on that export for now.